app/components/settings_form/settings_form.html.erb

```diff
@@ -67,7 +67,9 @@
 
     <% if current_user.admin? %>
       <%= c 'stack' do %>
+        <%= c('heading', tag: :h2) { t('setting.form.groups.all') } %>
         <%= c('settings_field', type: :image_input, attr: :channel_image) %>
+        <%= c('settings_field', type: :textarea, attr: :about) %>
```

Comment:
This also looks like unrelated code that could be merged in a separate PR.

Small and many PRs for the win!